### DISCUSSION TITLE
using Semantic Versionning

### DISCUSSION THREAD

**Discussion by makayabou** [docusealco/docuseal#264] [Ideas]
As presented at https://semver.org/, it would be great to use semantic versioning.

We have Docuseal 1.5.4 now, which introduces breaking change.
Breaking changes should provoke first number incrementation.
Using semver we would have named it 2.0.0

That way developers know better when they face a version problem after a breaking change in core or api.

What do you think ?

**Comment by omohokcoj** [comment]
@makayabou that's a very minor 'change' that might break things for a small % of users (that use Strong typed API response parsers and parse the values response JSON param - it used to be a {} object but now it's an [] array )
I don't think we need to issue a 2.0 just because of this change
